Regression/WarpX-tests.ini

```diff
@@ -1271,6 +1271,21 @@ tolerance = 1.e-14
 buildDir = .
```

Comment:
I think that we can remove the test `TwoParticle_electrostatic` altogether: as it is right now, this tests only runs the code but does not check the results. The `ElectrostaticSphere` test checks the same capabilities, but with some extra checks.